src/node_errors.h

```diff
@@ -19,9 +19,11 @@ void AppendExceptionLine(Environment* env,
                          v8::Local<v8::Message> message,
                          enum ErrorHandlingMode mode);
 
+// This function calls backtrace, it should have not be marked as [[noreturn]].
+// But it is a public API, removing the attribute can break.
 [[noreturn]] void OnFatalError(const char* location, const char* message);
-[[noreturn]] void OOMErrorHandler(const char* location,
-                                  const v8::OOMDetails& details);
+// This function calls backtrace, do not mark as [[noreturn]].
```

Comment:
Can you add a comment to here explaining why calling backtrace should not be marked as noreturn? For most contributors, the answer is not obvious.

Reply:
Thanks for the suggestion, updated!